mne/epochs.py

```diff
@@ -1325,7 +1325,8 @@ def _project_epoch(self, epoch):
         return epoch
 
     @verbose
-    def _get_data(self, out=True, picks=None, item=None, verbose=None):
+    def _get_data(self, out=True, picks=None, item=None, units=None,
```

Comment:
Might help ensure internally we don't make a mistake
```suggestion
    def _get_data(self, out=True, picks=None, item=None, *, units=None,
```